code/modules/religion/religion_sects.dm

```diff
@@ -213,7 +223,141 @@
+		playsound(user, "punch", 25, TRUE, -1)
+		SEND_SIGNAL(H, COMSIG_ADD_MOOD_EVENT, "blessing", /datum/mood_event/blessing)
+	return TRUE
+	*/
+
+/datum/religion_sect/honorbound
+	name = "Honorbound God"
+	desc = "A sect that does not engage in harm."
+	altar_icon_state = "convertaltar-white"
+	alignment = ALIGNMENT_GOOD
+	convert_opener = "\"A good, honourable crusade against evil is required.\"<br>\
+	Your deity requires fair fights from you. You may not attack the unready, the just, or the innocent.<br>\
+	You earn favor by getting others to join the crusade, and you may spend favor to announce a battle, bypassing some conditions to attack."
+
+/datum/religion_sect/honorbound/on_conversion(mob/living/carbon/new_convert)
+	if(!ishuman(new_convert))
+		to_chat("<span class='warning'>[GLOB.deity] has no respect for lower creatures, and refuses to make you honorbound.</span>")
+	else
+		var/datum/dna/holy_dna = new_convert.dna
+		holy_dna.add_mutation(/datum/mutation/human/honorbound)
+
+/datum/religion_sect/burden
+	name = "Punished God"
+	desc = "A sect that desires to feel the pain of their god."
+	altar_icon_state = "convertaltar-burden"
+	alignment = ALIGNMENT_NEUT
+	convert_opener = "\"To feel the freedom, you must first understand captivity.\"<br>\
+	Incapacitate yourself in any way possible. Bad mutations, lost limbs, traumas, even addictions. You will learn the secrets of the universe \
+	from your defeated shell."
+	//a list for keeping track of how burdened each member is
+
+/datum/religion_sect/burden/on_conversion(mob/living/carbon/new_convert)
+	if(!ishuman(new_convert))
+		to_chat("<span class='warning'>[GLOB.deity] needs higher level creatures to fully comprehend the suffering. You are not burdened.</span>")
+	else
+		var/datum/dna/holy_dna = new_convert.dna
+		holy_dna.add_mutation(/datum/mutation/human/burdened)
+
+/datum/religion_sect/burden/tool_examine(mob/living/carbon/human/burdened) //display money policy
+	var/datum/mutation/human/burdened/burdenmut = burdened.dna.check_mutation(/datum/mutation/human/burdened)
+	if(burdenmut)
+		return "<span class='notice'>You are at burden level [burdenmut.burden_level]/6.</span>"
+	return "<span class='notice'>You are not burdened.</span>"
+
+#define MINIMUM_YUCK_REQUIRED 5
+
+/datum/religion_sect/maintenance
+	name = "Maintenance God"
+	desc = "A sect based around the maintenance shafts of the station."
+	altar_icon_state = "convertaltar-maint"
+	alignment = ALIGNMENT_EVIL //while maint is more neutral in my eyes, the flavor of it kinda pertains to rotting and becoming corrupted by the maints
+	convert_opener = "\"Your kingdom in the darkness.\"<br>\
+	Sacrifice the organic slurry created from rats dipped in welding fuel to gain favor. Exchange favor to adapt to the maintenance shafts."
+	rites_list = list(/datum/religion_rites/maint_adaptation, /datum/religion_rites/adapted_food, /datum/religion_rites/ritual_totem)
+	desired_items = list(/obj/item/reagent_containers)
+
```

Comment:
maybe make adopting this sect add maint access to your ID card if you're wearing one?